From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 23 Sep 2019 22:06:39 +0100
Subject: PCI: Avoid ABI change in 3.16.74
Forwarded: not-needed

Commit 57b9ed53b9ad "PCI: Work around Pericom PCIe-to-PCI bridge
Retrain Link erratum" added a new member pci_dev::clear_retrain_link.
Thankfully this is a bitfield that replaces a padding bit and doesn't
otherwise change the structure layout.  Hide it from genksyms.

---
--- a/include/linux/pci.h
+++ b/include/linux/pci.h
@@ -308,8 +308,11 @@ struct pci_dev {
 						   powered on/off by the
 						   corresponding bridge */
 	unsigned int	ignore_hotplug:1;	/* Ignore hotplug events */
+#ifndef __GENKSYMS__
 	unsigned int	clear_retrain_link:1;	/* Need to clear Retrain Link
 						   bit manually */
+	/* 15 bits spare */
+#endif
 	unsigned int	d3_delay;	/* D3->D0 transition time in ms */
 	unsigned int	d3cold_delay;	/* D3cold->D0 transition time in ms */
 
